src/media/js/elements/nav.js

```diff
@@ -241,9 +241,11 @@ define('elements/nav',
         // wouldn't apply unless <html>/<body> had heights, and setting to
         // 100% height would mess up the page since our <footer> isn't part of
         // <main>. Could be resolved if we made <footer> a part of <main>.
-        var main = document.querySelector('main');
-        if (main) {
-            document.querySelector('main').style.minHeight = screen.height + 'px';
+        if (settings.mktNavEnabled) {
+            var main = document.querySelector('main');
+            if (main) {
+                main.style.minHeight = screen.height + 'px';
```

Comment:
I think I'd rather not use jQuery if we don't need to. This is pretty straightforward and avoiding having to add `+ 'px'` doesn't seem like a big benefit.